frontend/package.json

```diff
@@ -25,6 +25,8 @@
     "@ionic/react-hooks": "^0.0.8",
     "@ionic/react-router": "5.6.5",
     "aws-amplify-react": "^5.0.0",
+    "@types/dom-mediacapture-record": "^1.0.7",
+    "cordova-plugin-media-capture": "^3.0.3",
```

Comment:
Strange that Github sees the changes here. Since it is coming from #37. Could you rebase it again?

Reply:
Good catch. Something seemed to have gone wrong with the rebase. 
Is now fixed! Thanks for pointing out!